src/components/plots/barPlot.ts

```diff
@@ -315,6 +351,9 @@ export module Plots {
     }
 
     private _updateValueScale() {
+      if (!this._projectorsReady()) {
+        return;
+      }
```

Comment:
it is called by the constructor, through `baseline()`. Hence guard needed.
I would prefer not to do any code change anymore as that needs QE again.